style.css

```diff
@@ -127,6 +151,202 @@ button {
   margin-top: 8%;
 }
 
+/* boxes */
+.boxes {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-content: center;
+  margin-top: 25px;
+  gap: 2em;
+}
+
+.box {
+  height: 474px;
+  width: 327px;
+  background-color: #d0d9d4;
+  border-radius: 12px;
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  justify-content: space-between;
+}
+
+.box-menu {
+  background-color: #fff;
+  height: 230px;
+  border-bottom-left-radius: 12px;
+  border-bottom-right-radius: 12px;
+}
+
+.box-list {
+  display: flex;
+  flex-direction: row;
+  margin: 27px;
+  gap: 5px;
+}
+
+.box-list-item {
+  background-color: #d0d9d4;
+  padding: 3px;
+  border-radius: 3px;
+}
+
+.box-button {
+  color: #fff;
+  background: #36b37f;
+  width: 122px;
+  height: 48px;
+  margin-top: 7px;
+  border-radius: 5px;
+}
```

Comment:
- [ ] Dear @danutibe07, For your `See Project` button please add the button styling: `Active`, `Hover `, and `Pressed`

![Screenshot 2023-01-13 092215](https://user-images.githubusercontent.com/69077061/212251715-51b9cb02-9e18-426f-a7d9-8f9da8d8c72f.png)